src/Make/MakeBackupZip.php

```diff
@@ -18,9 +18,9 @@
  */
 class MakeBackupZip extends AbstractMakeZip
```

Comment:
This code appears to be abandoned, and not reachable (except from the MakeBackupZip test).

Reply:
Yeah, I need to make it live again from console script, that's why it's kept around.